packages/app-js/src/snippets/index.ts

```diff
@@ -0,0 +1,31 @@
+import { Snippet } from './../types';
```

Comment:
Header missing.

Reply:
You also took care of that in the PR, so I'm closing this.